docker-compose.personal-server.yaml

```diff
@@ -0,0 +1,57 @@
+---
+services:
+  app: &app_env
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    ports:
+      - 127.0.0.1:8080:8080
+    environment:
+      IS_DOCKER: '1'
+      FLASK_APP: hushline
+      FLASK_ENV: development
+      ENCRYPTION_KEY: bi5FDwhZGKfc4urLJ_ChGtIAaOPgxd3RDOhnvct10mw=
+      SECRET_KEY: cb3f4afde364bfb3956b97ca22ef4d2b593d9d980a4330686267cabcd2c0befd
+      SESSION_FERNET_KEY: jY0gDbATEOQolx2SGj46YnkkbN6HQBB4YCABzwl1H1A=
+      SQLALCHEMY_DATABASE_URI: postgresql://hushline:hushline@postgres:5432/hushline
+      REGISTRATION_CODES_REQUIRED: "false"
+      ALIAS_MODE: always
```

Comment:
Maybe we should have these as make targets and toggle the few env vars. Would that be sufficient?

```yaml
  FOO_VAR: ${FOO_VAR}
```